### DISCUSSION TITLE
how to get all address from wallet?

### DISCUSSION THREAD

**Discussion by ovicloud** [Chia-Network/chia-blockchain#6335] [Support]
how to get all address from wallet?  rpc interface "get_next_address" only create new address or get last address .

**Comment by BrandtH22** [comment]
@ovicloud , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - the chia wallet can have a nearly infinite number of possible addresses, to get lists of these you can use the `chia keys derive wallet-address -n <#_of_Addresses>` command to derive a set number of addresses. More information on this command can be found https://docs.chia.net/cli#wallet-address

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
